test/qunit/model/FHIRModel.unit.js

```diff
@@ -1311,6 +1311,16 @@ sap.ui.define([
 		mParameters = oListBinding._buildParameters();
 		oRequestHandle = oFhirModel.loadData("/Patient", mParameters);
 		assert.deepEqual(mParameters.urlParameters["_filter"], "( name sw \"Ra\" and name ew \"er\" )", "The _filter parameter for StartsWith and EndsWith operator is the formed correctly");
+
+		var oBirthDate = new Date("2014", "2", "2");
+		var sBirthDateISOString = oBirthDate.toISOString();
+		oBirthDateFilter = new FHIRFilter({ path: "birthdate", operator: FHIRFilterOperator.GT, value1: oBirthDate });
+		aFilters = [oBirthDateFilter];
+		oListBinding.filter(aFilters);
+		mParameters = oListBinding._buildParameters();
+		oRequestHandle = oFhirModel.loadData("/Patient", mParameters);
+		var sFilterParams = "birthdate gt " + sBirthDateISOString;
```

Comment:
IMHO using the same function `toISOString` here to generate the expected result is too risky.
I recommend to  use the hard coded value "2014-03-01T23:00:00.000Z". Is it FHIR compliant to have hours, minutes and seconds in the filter string?

Reply:
@flovogt : yes _filter string can have timezone.

Reply:
@flovogt : i cannot hardcode the value since when running the tests in local the iso value retuned is different than that of the server where the tests are run. so toIsoString is the better thing here

Reply:
The tests should not be bound too much on the browser setup.
Could you try following:
````

var oBirthDate = new Date(Date.UTC(2020, 3 ,12, 10, 0, 0));
````
And then using the hardcoded value in the assertion?

Reply:
@flovogt : It works! Done